Sources/KSCrashDiscSpaceMonitor/KSCrashMonitor_DiscSpace.m

```diff
@@ -69,28 +67,24 @@ static bool isEnabled(void)
 
 static void addContextualInfoToEvent(KSCrash_MonitorContext* eventContext)
 {
-    if(g_isEnabled)
+    if (g_isEnabled)
     {
         eventContext->System.storageSize = getStorageSize();
     }
 }
 
 KSCrashMonitorAPI* kscm_discspace_getAPI(void)
 {
-    static KSCrashMonitorAPI api =
-    {
-        .monitorId = monitorId,
-        .setEnabled = setEnabled,
-        .isEnabled = isEnabled,
-        .addContextualInfoToEvent = addContextualInfoToEvent
-    };
+    static KSCrashMonitorAPI api = { .monitorId = monitorId,
+                                     .setEnabled = setEnabled,
+                                     .isEnabled = isEnabled,
+                                     .addContextualInfoToEvent = addContextualInfoToEvent };
```

Comment:
The original version was better. Any chance to fix this?

Reply:
I couldn't get it to work with `clang-format'.

Reply:
That from LLVM standart

Reply:
And from Google's as well